package/python-kiwi-spec-template

```diff
@@ -215,7 +215,7 @@ Obsoletes:      kiwi-filesystem-requires < %{version}-%{release}
 %if "%{_vendor}" != "debbuild"
 Provides:       kiwi-image:pxe
 Provides:       kiwi-image:kis
-%if ! 0%{?el8}
+%if ! 0%{?el8} && ! 0%{?el9}
```

Comment:
How about `%if ! (0%{?rhel} >= 8)` instead? That's what I use in the Fedora kiwi spec.

Reply:
sounds good, how about it now ?